.gitignore

```
@@ -132,4 +132,4 @@ dmypy.json
 .pyre/
 
 # vscode
-.vscode/
+.vscode/
```

```suggestion
.vscode/

```
Adding the newline back at the end of the file so GitHub is happy

---

Ayy my bad! I messed around with Path variables. I've fixed them now along with the unzip=True line.